config/rapidez.php

```diff
@@ -63,6 +63,10 @@
         'block'        => Rapidez\Core\Models\Block::class,
     ],
 
+    'widgets' => [
+
```

Comment:
This will be for example:
`'JustBetter\MultiPurposeWidget\Block\Widget\Notification' => App\Widgets\NotificationWidget::class`